Clarify architecture overview (#4144)
Particular/docs.particular.net: platform/servicecontrol.include.md

> ServiceControl is a suite of back-end tools that collect useful information about a running NServiceBus system. It collects data on every single message flowing through the system (via an audit queue), errors (via an error queue), as well as additional information regarding sagas, endpoints [heartbeats](/monitoring/heartbeats/) and [custom checks](/monitoring/custom-checks/) (via a control queue). The information is then exposed to [ServicePulse](/servicepulse) and [ServiceInsight](/serviceinsight) through an HTTP API and SignalR notifications.

Reviewer: > ...through an HTTP API and SignalR notifications.

I'm not sure this is an improvement on the previous wording, but I can see why it's been done due to the addition of "via" in a few other places. I think a better solution is to restructure that other sentence, which reads quite awkwardly. Perhaps:

```suggestion
ServiceControl is a background process that collects useful information about an NServiceBus system. It uses an audit queue to collect every message flowing through the system and an error queue to collect failed messages. It can also collect saga state changes, endpoint [heartbeats](/monitoring/heartbeats/), and perform [custom checks](/monitoring/custom-checks/) using a control queue. This information is exposed to [ServicePulse](/servicepulse) and [ServiceInsight](/serviceinsight) via an HTTP API and SignalR notifications.
```

NOTE: ServiceControl _consumes_ messages that arrive in either the configured audit or error queues, i.e. it removes those messages from the queues. If a copy of those messages is required for further processing, configure [audit forwarding](/servicecontrol/creating-config-file.md#transport-servicecontrolforwardauditmessages) and/or [error queue forwarding](/servicecontrol/creating-config-file.md#transport-servicecontrolforwarderrormessages).

By default ServiceControl stores information for 30 days, but this period can be [customized](/servicecontrol/creating-config-file.md).

> Refer to the [Optimizing For Use In Different Environments](/servicecontrol/servicecontrol-in-practice.md) article for more information about practical considerations.

Reviewer: If the title of the page is being used, it should _be_ the title of the page:

```suggestion
See [Optimizing ServiceControl for use in different environments](/servicecontrol/servicecontrol-in-practice.md) for more information about practical considerations.
```
